libs/clients/auth/admin-api/src/lib/auth-admin-api-client.config.ts

```diff
@@ -0,0 +1,25 @@
+import * as z from 'zod'
+
+import { defineConfig } from '@island.is/nest/config'
+
+const schema = z.object({
+  basePaths: z.object({
+    dev: z.string().optional(),
+    staging: z.string().optional(),
+    prod: z.string().optional(),
+  }),
+})
+
+type BasePaths = z.infer<typeof schema>['basePaths']
+
+export const AuthAdminApiClientConfig = defineConfig({
+  name: 'AuthAdminApiClientConfig',
+  schema,
+  load(env) {
+    return {
+      basePaths: env.requiredJSON<BasePaths>('AUTH_ADMIN_API_PATHS', {
+        dev: 'http://localhost:6333',
+      }),
```

Comment:
Huh. Did you need to specify the generics argument here?

Reply:
Nope, TS in webstorm was messing with me.